src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @OneToOne
```

Comment:
```suggestion
    @ManyToOne
```

Reply:
but if look at this picture, we can see that there are relations OneToOne between MovieSession and Movie
![Hibernate_Cinema_Uml](https://github.com/user-attachments/assets/7293c4e4-86e0-468f-8e7f-aa93dc2a91b6)